english/Harpers/yourlife.md

```diff
@@ -0,0 +1,5 @@
+This is the beginning of your life. Make wise choices to cultivate a healthy body
+and a healthy mind.
+
+click [here](https://github.com/udacity/create-your-own-adventure/blob/master/Polish/przygoda/jedzenie/akap/drzwi/drzwi.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.